src/Microsoft.Health.Fhir.Core/Features/Operations/DataConvert/Models/DataConvertTimeoutException.cs

```diff
@@ -0,0 +1,38 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using EnsureThat;
+using Microsoft.Health.Fhir.Core.Exceptions;
+using Microsoft.Health.Fhir.Core.Models;
+
+namespace Microsoft.Health.Fhir.Core.Features.Operations.DataConvert.Models
+{
+    public class DataConvertTimeoutException : FhirException
+    {
+        public DataConvertTimeoutException()
```

Comment:
Do we want to allow a blank constructor for this exception type? The other two generate an issue to return to the client, but this one doesn't add any extra information.

Reply:
we don't want a default constructor. I just add this following a code intelligence tool. I have removed the empty constructor.